eng/Analyzers.props

```diff
@@ -6,8 +6,8 @@
   </PropertyGroup>
   <ItemGroup Condition="'$(RunAnalyzers)' != 'false'">
     <PackageReference Include="Microsoft.DotNet.CodeAnalysis" Version="$(MicrosoftDotNetCodeAnalysisVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
-    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="6.0.0-preview1.20513.4" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeStyle" Version="3.8.0-4.20469.1" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="56.0.0-preview1.20513.4" PrivateAssets="all" />
```

Comment:
IS this version correct? I would be surprised if our version was 56, so seems like a typo 😄

Reply:
It seems my notification was not updated and pointed me to this change which is outdated 😄